api/package.json

```diff
@@ -1,13 +1,19 @@
 {
-  "name": "react-app-with-json-server",
+  "name": "json-server",
   "version": "1.0.0",
   "engines": {
     "node": ">=18.0.0"
   },
   "scripts": {
-    "start": "npm i && node server.js"
+    "build": "tsc",
+    "start": "node server.js"
   },
   "dependencies": {
+    "cors": "^2.8.5",
```

Comment:
```suggestion
```
As far as I know we don't actually use it in the `api/server.ts` file

Reply:
You are right you added it but then we actually did not use it, I removed it.